discord/message.py

```diff
@@ -628,7 +649,7 @@ def _add_reaction(self, data, emoji, user_id):
 
         return reaction
 
-    def _remove_reaction(self, data, emoji, user_id):
+    def _remove_reaction(self, data: ReactionPayload, emoji: Union[Emoji, PartialEmoji, str], user_id: int) -> Reaction:
```

Comment:
I think `Union[Emoji, PartialEmoji, str]` should probably be aliased since it's used throughout the file. Something like `EmojiInputType` or something within the `TYPE_CHECKING` block.